.gitignore

```diff
@@ -188,3 +188,7 @@ src/Orchard.Azure/Orchard.Azure.CloudService/Staging/
 #enable all /lib artifacts
 !lib/*/*.*
 */.vs/*
+src/Orchard.Web/Modules/Naxos*
```

Comment:
I think I see the problem and it's my in-experience with using GitHub and pull requests.  The branch I would like a pull request from is feature/Content-Picker-MultiSelect not the Naxos branch as the Naxos branch has our specific code changes and as you see git ignore so I can separate our code from the orchard code.